packages/flutter/test/widgets/scroll_notification_test.dart

```diff
@@ -45,13 +45,13 @@ void main() {
     await gesture.up();
     await tester.pump(const Duration(seconds: 1));
 
-    expect(events.length, 5);
-    // user scroll do not trigger the ScrollMetricsNotification.
+    expect(events.length, 6);
     expect(events[0] is ScrollStartNotification, true);
     expect(events[1] is UserScrollNotification, true);
     expect(events[2] is ScrollUpdateNotification, true);
-    expect(events[3] is ScrollEndNotification, true);
-    expect(events[4] is UserScrollNotification, true);
+    expect(events[3] is ScrollMetricsNotification, true);
```

Comment:
I just catch this now, but if only pixel (scroll position) is changing shouldn't this be `ScrollUpdateNotification` ? in that case line 51 has already included that, do we need to fire another ScrollMetricsNotification?